readium/shared/src/main/java/org/readium/r2/shared/publication/Metadata.kt

```diff
@@ -65,6 +66,7 @@ data class Metadata(
     constructor(
         identifier: String? = null, // URI
         type: String? = null, // URI (@type)
+        conformsTo: List<Publication.Profile> = emptyList(),
```

Comment:
I would use a `Set` since the order is not meaningful.

Reply:
Good point, let me fix that.